core/src/main/java/org/geysermc/geyser/network/PacketCooldownManager.java

```diff
@@ -0,0 +1,100 @@
+package org.geysermc.geyser.network;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.cloudburstmc.protocol.bedrock.packet.*;
+import org.geysermc.geyser.session.GeyserSession;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class PacketCooldownManager {
+    private final Map<String, CooldownSettings> packetCooldownSettings = new HashMap<>();
+
+    private final GeyserSession session;
+    @Setter
+    private long cooldownMillisDebug;
+    private long expiryTimeMillisDebug;
+
+    public PacketCooldownManager(GeyserSession session, long cooldownMillisDebug) {
+        this.session = session;
+        this.setCooldownMillisDebug(cooldownMillisDebug);
+        this.expiryTimeMillisDebug = 0;
+
+        setPacketCooldown(LoginPacket.class, -1, 2);
+        setPacketCooldown(ResourcePackClientResponsePacket.class, -1, 4);
+        setPacketCooldown(ResourcePackChunkRequestPacket.class, -1, 0);
+        setPacketCooldown(TextPacket.class, 1000, 50);
+        setPacketCooldown(CommandRequestPacket.class, 1000, 50);
+        setPacketCooldown(ModalFormResponsePacket.class, 1000, 50);
```

Comment:
> Как были выбраны эти ценности?

I wanted to initially write messages that they violate without kicking and so that they could not spam the console, what are we going to do immediately kick?